packages/components/data-table-manager/src/data-table-settings/messages.js

```diff
@@ -9,7 +9,7 @@ export default defineMessages({
   displaySettingsOption: {
     id: 'UIKit.DataTableManager.TableSettings.displaySettingsOption',
     description: 'Display settings dropdown option label.',
-    defaultMessage: 'Display settings',
+    defaultMessage: 'Layout settings',
```

Comment:
Btw, this will cause the translations to be removed, since the source file changed.

Reply:
Ah, I see. I'll just leave these untouched then.